src/editors/collectionDocumentsProvider.ts

```diff
@@ -99,7 +98,7 @@ export default class CollectionViewProvider implements vscode.TextDocumentConten
             operation.hasMoreDocumentsToShow = false;
           }
 
-          return resolve(EJSON.stringify(documents, null, 2));
+          return resolve(JSON.stringify(documents, null, 2));
```

Comment:
Why not EJSON?

Reply:
I think this is something we might want to revisit when we work on editing documents. Chatted with @mmarcon and we were thinking JSON looks a little more intuitive to users.
Having objectId written as `5e32b4d67bf47f4525f2f8ab` vs
```json
{
  "$oid": "5e32b4d67bf47f4525f2f8ab"
}
```
for example.

Reply:
I think best case might be to write output like `_id: ObjectId('5e32b4d67bf47f4525f2f8ab')` but we'll need to not display results in a json document then, and instead have a different output format...

Reply:
Will be `JSON.stringify()` able to handle any BSON document without throwing errors?

Reply:
Yup - added a test for it.